corehq/apps/ota/views.py

```diff
@@ -106,6 +107,8 @@ def search(request, domain):
 
     try:
         hits = search_es.run().raw_hits
+    except TooManyRelatedCasesException:
```

Comment:
I think this will get raised from a few lines above: `CaseSearchCriteria(domain, case_type, criteria)` since that's where the query is constructed which is when the parent case ID query is run.

Reply:
I think you should also handle the `UnsupportedQueryException`

Reply:
1ee2b33e3d14737608ce6d2c28c6b094a0d5ebae and 4bd7074138cf88edb2eeb449bee3808c74e53fbc